README.md

```diff
@@ -71,14 +71,12 @@ For user convenience, this section assumes that users will deploy the infrastruc
       
 1. Access the JupyterLab environment
    Open `localhost:8888` in your web browser. The default token for the JupyterLab is `123`
-1. Write your first example program
-   In the JupyterLab, create a new file, `program.py`, in the `work` directory.
-   
-   Save [this example python code](https://qiskit-extensions.github.io/quantum-serverless/quickstart/index.html#id8).  
+1. Write your first example Qiskit Pattern.
+   In the JupyterLab, create a new file, `pattern.py`, in the `work` directory. You can include any arbitrary Python code in your program, or you can use the
+   [example Python file in this tutorial](https://github.com/caleb-johnson/quantum-serverless/blob/main/docs/getting_started/basic/01_running_program.ipynb).
 
-   Now, you are ready to run the first program.
 1. Run the program
-   In the JupyterLab, create a new notebook in the same directory as your program, and execute [this python code](https://qiskit-extensions.github.io/quantum-serverless/quickstart/index.html#id9).
+   In the JupyterLab, create a new notebook in the same directory as your program, and execute [the tutorial code](https://github.com/caleb-johnson/quantum-serverless/blob/main/docs/getting_started/basic/01_running_program.ipynb).
```

Comment:
`https://github.com/caleb-johnson/q` ? 😄

Reply:
lol I'm trying to drive up traffic to my fork ;)

Nice catch